include/godot_cpp/classes/wrapped.hpp

```diff
@@ -408,7 +408,7 @@ protected:
 // Don't use this for your classes, use GDCLASS() instead.
 #define GDEXTENSION_CLASS_ALIAS(m_class, m_alias_for, m_inherits) /******************************************************************************************************************/ \
 private:                                                                                                                                                                               \
-	inline static ::godot::internal::EngineClassRegistration<m_class> _gde_engine_class_registration_helper;                                                                           \
+	static ::godot::internal::EngineClassRegistration<m_class> _gde_engine_class_registration_helper;                                                                                  \
```

Comment:
The `inline static` here makes is so that we can declare a static variable in a header without needing to setup its storage in a compilation unit. I think removing the `inline` here means here means this variable won't actually be initialized (and hence the registration won't happen) because we don't ever put it in a .cpp file. We could certainly generate an addition to the corresponding .cpp file, but then the registration will always happen for all classes, and not just those that were `#include`d.

Reply:
That is incorrect according to the disassembly and actually running the code...    This is the offending line that causes bloat because the linker isn't able to optimize all these templated unused symbols out.  The variable is initialized during static initialization when the library is loaded.

Reply:
`static` class variables (without `inline`) need to have their storage declared in a compilation unit (`.cpp` file) somewhere, otherwise they will never be initialized.

This PR is removing the `inline` but not adding the storage bit to any `.cpp` file.